src/extension.ts

```diff
@@ -493,12 +485,6 @@ function registerCommands(context: VSCode.ExtensionContext) {
 
   context.subscriptions.push(VSCode.commands.registerCommand(Commands.INSTALL_MANAGED_JRE, installManagedJre));
 
-  context.subscriptions.push(
```

Comment:
This command was used for `open hotspot in IDE` when we used to construct a hotspot ourselves.